### PR TITLE
source_map should have relative paths

### PR DESCRIPTION
**Pull request by bit**
prefix the source_map with the prefix of the source
to produce usable .map files.

### CODE REVIEW

**Comment by bit** [comment]
this change as the advantage that the tests now work independent of the directory depth of the source repository. previously this was hard-coded.

File: sass_processor/processor.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def __call__(self, path):
         if not self.processor_enabled:
             return css_filename
         sourcemap_filename = css_filename + '.map'
-        if find_file(css_filename) and self.is_latest(sourcemap_filename):
+        if find_file(css_filename) and self.is_latest(sourcemap_filename, filename):
```
Comment (jrief) [inline sass_processor/processor.py]: for better readability, I prefer a more verbose name for `filename`. The variable should explain, what kind of file this is.

Comment (bit) [reply sass_processor/processor.py]: would sourcefile be an ok name?

Comment (bit) [reply sass_processor/processor.py]: I have a better idea, I will rework it to pass a base folder to is_latest

**Comment by jrief** [comment]
does this solve the problem with sourcemaps not working properly on Windows? There the absolute name starts with, say ``C:\...``, hence SASS processor could not compute the proper name.

Anyway, thanks, even though I never ran into any problem with this up to now?

**Comment by bit** [comment]
Don't have windows around to test but should also make the situation on windows better.

**Comment by jrief** [comment]
Thanks

**Pull request merged**